packages/evm/src/opcodes/functions.ts

```diff
@@ -762,7 +762,7 @@ export const handlers: Map<number, OpHandler> = new Map([
     function (runState) {
       const index = runState.stack.pop()
       if (runState.env.blobVersionedHashes.length > Number(index)) {
-        runState.stack.push(bytesToBigInt(runState.env.blobVersionedHashes[Number(index)]))
+        runState.stack.push(BigInt(runState.env.blobVersionedHashes[Number(index)]))
```

Comment:
I will address this at some point, bit inside the EVM we should likely not put the strings in, but directly the bigints so we can push those to stack.

Reply:
Oh, hmm ok originally they were bytes which we kept converting to bigints, which is/was also suboptimal.